src/sdk/main/src/CustomRoyaltyFee.cc

```diff
@@ -91,11 +91,6 @@ CustomRoyaltyFee& CustomRoyaltyFee::setNumerator(const int64_t& numerator)
 //-----
 CustomRoyaltyFee& CustomRoyaltyFee::setDenominator(const int64_t& denominator)
 {
-  if (denominator == 0LL)
```

Comment:
Are these removed for a reason? Guessing it is checked service side?

Reply:
Yes exactly